### ISSUE TITLE
When Preview, shows black and said "Start a stream to enable the preview!"

### ISSUE THREAD

**Issue by hs79hs** [fyhertz/spydroid-ipcamera#21]
I installed Spydroid v8.0.2, using a mobile running Android 5.1, 
Cannot access the camera no matter using http or rtsp by vlc.

When Preview, it shows black and said "Start a stream to enable the preview!"
Has anyone have the same issue and what's the solution to this? Thanks.

**Comment by Hassaram** [comment]
Hi. I have the same issue. Was this resolved?

**Comment by lingjiaoling** [comment]
no. we chose another way. 发自网易邮箱大师 On 2016-11-09 00:04 , Hassaram Wrote: Hi. I have the same issue. Was this resolved? — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by kungercn** [comment]
you need use chrome open http url or vlc open rtsp url to connect android server

**Comment by FarhanQayyum123** [comment]
open app and u will see the IP address on first screen of app and then paste that IP address on chrome and then go to PREVIEW in app and then press CONNECT button on chrome and then u will see the preview, i am doing the same but not playing in chrome, anyone can resolve my issue plzz..???

**Comment by ngethe** [comment]
Please help I have the same issues

**Comment by rovaals** [comment]
I have the same problem and can see in alogcat that the framerate used is not supported. Is have an unusual device and I know from issues with other camera apps that it only supports 30fps, which it also states in the alogcat error when using spydroid. Anything else fails, and this app only goes up to 20fps. My device is rooted so I plan on trying to find the prefs file in the data folder and see if I can force it there.

**Comment by webhat** [comment]
I have the same issue, is this abandoned?
